packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -1,127 +1,128 @@
-.comment {
+.kt-comment {
 	display: flex;
 	flex-flow: row;
 	padding: 0.4rem 0;
 	border-bottom: 1px solid #dbdbdb;
-}
-
-.comment__avatar {
-	flex: 0 0 2.4rem;
-	height: 2.4rem;
-}
-
-.comment__wrapper {
-	display: flex;
-	flex: 1 1 auto;
-	flex-direction: column;
-	margin-left: 0.4rem;
-}
-
-.comment__info {
-	display: flex;
-	flex-direction: row;
-	width: 100%;
-	font-size: 0.7rem;
-	line-height: 1.2rem;
-}
-
-.info__name {
-	flex: 1 1;
-	font-weight: 600;
-}
-
-.info__time {
-	flex: 1 1;
-	color: $lightgray-500;
-	text-align: right;
-}
-
-.comment__message {
-	font-size: 0.75rem;
-	line-height: 1.2rem;
-	word-break: break-word;
-}
-
-.comment__action,
-.comment-reply__action {
-	display: flex;
-	flex-direction: row;
-	justify-content: space-between;
-	margin: 0.2rem 0;
-	font-size: 0.7rem;
-	font-weight: 600;
-	line-height: 1.2rem;
-	color: $lightgray-500;
-	.yoco {
-		font-size: 0.9rem;
+	&__wrapper {
+		display: flex;
+		flex: 1 1 auto;
+		flex-direction: column;
+		margin-left: 0.4rem;
+	}
+	&__info {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		font-size: 0.7rem;
+		line-height: 1.2rem;
+		&__name {
+			flex: 1 1;
+			font-weight: 600;
+		}
+		&__time {
+			flex: 1 1;
```

Comment:
In my experience, 90% of cases where `flex: 1 y z` is used can be replaced by `flex: 1`. Not sure if possible here, but easy to try out